doc/WebSite/Feature-Management.md

```diff
@@ -212,31 +212,31 @@ Used to simply check if given feature is enabled or not. Example:
         ...
     }
 
-IsEnabledAsync and other methods have also sync versions.
+The IsEnabledAsync and other methods also have sync versions.
 
-Surely, IsEnabled method should be used for **boolean type features**.
-Otherwise, you may get exceptions.
+The IsEnabled method should be used for **boolean type features**,
+otherwise you may get exceptions.
 
-If you just want to check a feature and throw exception as shown in the
-example, you can just use **CheckEnabled** method.
+If you just want to check a feature and throw an exception as shown in the
+example, you can just use the **CheckEnabled** method.
 
 ##### GetValue
 
-Used to get current value of a feature for value type features. Example:
+Used to get the current value of a feature for value-type features. Example:
 
     var createdTaskCountInThisMonth = GetCreatedTaskCountInThisMonth();
     if (createdTaskCountInThisMonth >= FeatureChecker.GetValue("MaxTaskCreationLimitPerMonth").To<int>())
     {
         throw new AbpAuthorizationException("You exceed task creation limit for this month, sorry :(");
     }
 
-FeatureChecker methods have also overrides to work features for a
-specified tenantId, not only for the current tenantId.
+The FeatureChecker methods also have overrides so that features can not only work for the
```

Comment:
"The FeatureChecker methods also have overrides ~`so that`~ `to check` features ~`can`~ not only ~`work`~ for the" is closer to the original meaning.